src/main/java/Main.java

```diff
@@ -0,0 +1,19 @@
+import com.leff.midi.MidiFile;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+public class Main {
+    public static void main(String[] args) {
+        try {
+            byte[] midiData = Files.readAllBytes(Paths.get("example.mid"));
```

Comment:
Read's the entirety of the Midi file into a `byte[]` at the top level so that it will always be visible to the debugger.